data/pl.json

```diff
@@ -0,0 +1,487 @@
+                "text": "do 10 sekund",
+                "icon": "slow_motion_video"
+            },
+            {
+                "name": "Tworzenie grup i kanałów",
+                "hint": "",
+                "text": "do 50 dziennie",
+                "icon": "add"
+            },
+            {
+                "name": "Ilość kont",
+                "hint": "(oficjalni klienci)",
+                "text": "do 3",
+                "icon": "group_work"
+            }
+        ]
+    },
+    {
+        "name": "Czaty i grupy",
+        "icon": "forum",
+        "color": "244, 64, 52",
+        "items": [
+            {
+                "name": "Liczba członków",
+                "hint": "(limit stale rośnie)",
+                "text": "do 200 000",
+                "icon": "group_add"
+            },
+            {
+                "name": "Liczba Administratorów",
+                "hint": "w grupie",
+                "text": "do 50",
+                "icon": "security"
+            },
+            {
+                "name": "Liczba botów",
+                "hint": "w grupie",
+                "text": "do 20",
+                "icon": "settings"
+            },
+            {
+                "name": "Przypięte czaty na głównej liście",
+                "hint": "",
+                "text": "do 5 czatów lub kanałów + 5 szyfrowanych czatów",
+                "icon": "push_pin"
+            },
+            {
+                "name": "Nazwa i opis grupy",
+                "hint": "",
+                "text": "do 255 znaków",
+                "icon": "subject"
+            },
+            {
+                "name": "Edycja wiadomości",
+                "hint": "",
+                "text": "w ciągu 48 godzin od wysłania; nieograniczona w Zapisane wiadomości i dla administratorów z prawami przypinania",
+                "icon": "edit"
+            },
+            {
+                "name": "Samozniszczenie zdjęć i filmów",
+                "hint": "",
+                "text": "1 do 60 sekund",
+                "icon": "shutter_speed"
+            },
+            {
+                "name": "Ostatnie zdarzenia",
+                "hint": "(subskrypcje, posty itp.)",
+                "text": "ostatnie 2 dni",
+                "icon": "history"
+            },
+            {
+                "name": "Liczba wzmianek",
+                "hint": "w jednej wiadomości",
+                "text": "do 50, w przeciwnym razie nie będzie powiadomienia",
+                "icon": "alternate_email"
+            },
+            {
+                "name": "Liczba widocznych wiadomości",
+                "hint": "w grupie",
+                "text": "1 000 000 ostatnich wiadomości",
+                "icon": "visibility"
+            },
+            {
+                "name": "Liczba subskrybentów, aby zobaczyć statystyki",
+                "hint": "",
+                "text": "500 i więcej",
+                "icon": "poll"
+            },
+            {
```

Comment:
I guess you started translating before some updates to limits were introduced
Translate and add the parts introduced in c0c7461

Reply:
Done